src/App.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from 'react';
 import Dropdown from './components/Dropdown';
-import Chart from './components/Chart';
+import BarChart from './components/BarChart';
 import './App.css';
 
 class App extends Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
     this.state = {
-      items: [],
+      covidData: [],
       isLoaded: false,
       location: '',
+      barChartData: {},
     };
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
```

Comment:
It doesn't look like you need a constructor here. You can assign default state values on the `App` component like so without the constructor. For `handleChange` and `handleSubmit`, as long as you use an arrow function then you do not need to bind.
```
state = {
    covidData: [],
    isLoaded : false,
    location: '',
    barChartData: {},
}
```